.github/release.yml

```diff
@@ -9,15 +9,20 @@ changelog:
     - title: New Features 🎉
       labels:
         - kind/feature
+        - kind/feature-request
     - title: Bugfixes 🐛
       labels:
         - kind/bug
     - title: Other Changes
       labels:
         - kind/other
+        - kind/user-story
```

Comment:
This was not added on Correspondence,  either remove here or add on correspondence aswell

Reply:
Removed